### PR TITLE
Update `package.json` dependencies

### PR DESCRIPTION
**Pull request by kienstra**
#### Changes
* Update `package.json` dependencies
* Including replacing `node-sass` with `sass`
* Fix linting and tests for those updates

#### Testing instructions
* Just a sanity-check that no code is obviously out of place

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -38,8 +30,7 @@ jobs:
       name: php
     steps:
       - checkout
-      - node/install:
-          node-version: '14.18.1'
+      - node/install
```
Comment (kienstra) [inline .circleci/config.yml:33]: It looks like the orb now finds the Node version from `.nvmrc`

File: .eslintrc [hunk 2]
```diff
@@ -5,7 +5,6 @@
     "plugin:import/recommended",
     "plugin:eslint-comments/recommended",
     "plugin:jsx-a11y/recommended",
-    "plugin:testing-library/recommended",
```
Comment (kienstra) [inline .eslintrc:8]: It sounds like this doesn't exist anymore

File: .circleci/config.yml [hunk 3]
```diff
@@ -18,15 +19,6 @@ commands:
...
 
-  install-composer:
-    description: "Installing Composer"
-    steps:
-      - run:
-          command: |
-            php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
-            EXPECTED_SIGNATURE=$(curl -s https://composer.github.io/installer.sig)
-            ACTUAL_SIGNATURE=$(php -r "echo hash_file('sha384', 'composer-setup.php');")
-            [[ "$EXPECTED_SIGNATURE" == "$ACTUAL_SIGNATURE" ]] && sudo php composer-setup.php --install-dir=/bin --filename=composer || exit 1
```
Comment (kienstra) [inline .circleci/config.yml:29]: Not related to this PR, just a simplification

File: babel.config.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
```
Comment (kienstra) [inline babel.config.js:1]: To replace `.babelrc`

File: css/src/_color.scss [hunk 5]
```diff
@@ -1,35 +1,28 @@
...
 
-// Hugo's new WordPress shades of gray, from http://codepen.io/hugobaeta/pen/grJjVp.
+/* Hugo's new WordPress shades of gray, from http://codepen.io/hugobaeta/pen/grJjVp. */
 $black: #000;
 $dark-gray-900: #191e23;
 $dark-gray-800: #23282d;
 $dark-gray-700: #32373c;
 $dark-gray-600: #40464d;
-$dark-gray-500: #555d66; // Use this most of the time for dark items.
+$dark-gray-500: #555d66;
```
Comment (kienstra) [inline css/src/_color.scss:9]: To address new linting rules. I'm not sure why this isn't allowed.

**Review by dreamwhisper — APPROVED** [review]

**Comment by kienstra** [comment]
Thanks a lot, @dreamwhisper!

**Pull request merged**